pop_transition/window.py

```diff
@@ -140,6 +141,99 @@ def current_page(self, page):
         }
         pages[page]()
     
+    def show_error(self, title:str, exception:Exception, package=None) -> None:
+        """Show an error dialog"""
+        self.error = ErrorDialog(self, title, exception)
+        message_text = traceback.format_exception_only(exception)[0].strip()
+        message_text, app_quit = self.parse_errors(
+            message_text, 
+            self.error, 
+            package=package
+        )
+        self.error.dialog_message.set_markup(message_text)
+        self.error.run()
+        self.error.destroy()
+        if app_quit:
+            self.quit_app()
+    
+    def parse_errors(self, message:str, dialog:Gtk.Dialog, package):
```

Comment:
Will this fail to recognize anything on non-English locales?

Reply:
@ids1024 I don't think so. In my testing and research, Python tracebacks/exceptions aren't localized, and since we're parsing the error based off of the exception message it should match either way.